src/views/certificate-signed.njk

```diff
@@ -9,26 +9,28 @@
       <div class="govuk-grid-column-two-thirds">
         <div class="interruption-card">
           <h1 id="blue-interruption-card" class="govuk-heading-xl govuk-!-margin-bottom-8">Application signed</h1>
-          <p class="govuk-body govuk-!-font-size-36 govuk-!-margin-bottom-1">Check the application's progress with the person who is submitting it</p>
+          <p class="govuk-body govuk-!-font-size-36 govuk-!-margin-bottom-1">The person making the application should notify you once it has been submitted.</p>
         </div>
       </div>
     </div>
 
     <div class="govuk-grid-row govuk-!-margin-top-4">
       <div class="govuk-grid-column-two-thirds">
 
-        <p id="signatures" class="govuk-body">We need to receive signatures from all signing {{ officerType }}s before the person making the application can pay for and submit it. It may take some time for others to sign.</p>
```

Comment:
The AC 2 says: The content for the type of company LLP, LTD or PLC shows the correct type. It looks like your change prevents it